src/alignfunctor.cpp

```diff
@@ -655,11 +655,8 @@ FunctorCode AlignVerticallyFunctor::VisitStaff(Staff *staff)
             const int verseNumber = verse->GetN();
             const data_STAFFREL versePlace = verse->GetPlace();
             const bool verseCollapse = m_doc->GetOptions()->m_lyricVerseCollapse.GetValue();
-            if ((versePlace == STAFFREL_above) && !alignment->GetVersePositionAbove(verseNumber, verseCollapse)) {
-                alignment->AddVerseN(verseNumber, verse->GetPlace());
-            }
-            if ((versePlace != STAFFREL_above) && !alignment->GetVersePositionBelow(verseNumber, verseCollapse)) {
-                alignment->AddVerseN(verseNumber, verse->GetPlace());
+            if (!alignment->GetVersePositionAbove(verseNumber, verseCollapse)) {
```

Comment:
Ah, sorry -- just re-looking. One is `GetVersePositionAbove` and one is `GetVersePositionBelow`...

Reply:
Yes, it was actually correct. Reverted.